zone/corpse.cpp

```diff
@@ -854,12 +883,18 @@ bool Corpse::IsEmpty() const {
 }
 
 bool Corpse::Process() {
-	if (player_corpse_depop)
+	if (player_corpse_depop) {
 		return false;
+	}
+
+	if (owner_online_timer.Check() || rezzable) {
```

Comment:
Is the `|| rezzable` here intentional? This could be pretty expensive polling for online characters from Corpse::Process. Maybe something can be done in world for client connect/disconnect to tell zones (and new booted zones) a player's online state instead of polling.

Reply:
Agreed. This is being needlessly polled when we can hook into existing places in the code that clearly can inform that connected status of a client

For example, where these events are triggered we can send a packet to all zones to inform status: EVENT_CONNECT or EVENT_DISCONNECT